docs/send-data/hosted-collectors/cloud-to-cloud-integration-framework/symantec-web-security-service-source.md

```diff
@@ -111,20 +111,22 @@ When Sumo Logic detects an issue it is tracked by Health Events. The following t
 
 ### JSON configuration
 
-Sources can be configured using UTF-8 encoded JSON files with the Collector Management API. See [how to use JSON to configure Sources](/docs/send-data/use-json-configure-sources) for details. 
+Sources can be configured using UTF-8 encoded JSON files with the [Collector Management API](/docs/api/collectors). See [how to use JSON to configure Sources](/docs/send-data/use-json-configure-sources) for details. 
 
 | Parameter | Type | Required | Description | Access |
 |--|--|--|--|--|
-| config | JSON Object  | Yes | Contains the configuration parameters for the Source. |   |
+| config | JSON Object  | Yes | Contains the [configuration parameters](/docs/send-data/hosted-collectors/cloud-to-cloud-integration-framework/symantec-web-security-service-source#config-parameters) for the Source. |   |
```

Comment:
You can just use the anchor, since it's a link within the same page
```suggestion
| config | JSON Object  | Yes | Contains the [configuration parameters](#config-parameters) for the Source. |   |
```